app/src/app/session/email-verification.dialog.html

```diff
@@ -0,0 +1,11 @@
+  <div [ngSwitch]="emailSent">
+    <div *ngSwitchCase="true">
+      A verification email has been sent to your email address
+    </div>
+    <div *ngSwitchCase="false">
+      An email has already been sent to your email, please check again
+    </div>
+    <div class="formactions">
+      <button mat-dialog-close  i18n>Ok</button>
+    </div>
```

Comment:
If everything is in the div started at line 1, then this div is useless because every component has a corresponding element in the DOM. You should must the div.formactions outside the div started at line 1. Moreover, there is not enough vertical space between the message and the buttons. As in DeletePollDialog, the div containing the message should have class dialog-message.